learn/cloud/analytics.mdx

```diff
@@ -0,0 +1,197 @@
+
+<Capsule intent="note" title="Analytics and latency">
+To capture and process search analytics data, Meilisearch Cloud must route API requests through a proxy. This might increase latency in some applications.
+</Capsule>
+
+### Configuring click-through rate and average click position
+
+To track metrics like clickthrough rate and average click position, Meilisearch Cloud needs to know when users click on search results.
+
+Every time a user clicks on a search result, your application must send a `click` event to the `POST` endpoint of Meilisearch Cloud analytics route:
+
+```sh
+curl \
+  -X POST 'http://edge.meilisearch.com' \
+  -H 'Content-Type: application/json' \
+  -H 'Authorization: Bearer Meilisearch_API_Key'
+  --data-binary '{ 
+    "eventType": "click",
+    "eventName": "search result click",
+    "indexUid": "products",
+    "objectID": "0",
+    "position": 0
+  }'
+```
+
+#### The `click` event object
+
+The `click` event must deliver an object with the following fields:
+
+```json
+{
+  "eventType": "click",
+  "eventName": "search result click",
+  "indexUid": "products",
+  "objectID": "0",
+  "position": 0
+}
+```
+
+- `eventType`: a string indicating this is a `click` event
+- `eventName`: a string describing the event
+- `indexUid`: a string indicating the clicked document's index
+- `objectID`: a string indicating the clicked document's primary key
+- `position`: an integer indicating the clicked document's position in the list of search results
+
+### Configuring conversion rate
+
+To track conversion rate, first identify what should count as a conversion for your application. For example, in a web shop, a conversion might be a user finalizing the checkout process. 
+
+Once you have established what counts as a conversion in your application, configure it to send a `conversion` event to the `POST` endpoint of Meilisearch Cloud analytics route:
+
+```sh
+curl \
+  -X POST 'http://edge.meilisearch.com' \
+  -H 'Content-Type: application/json' \
+  -H 'Authorization: Bearer Meilisearch_API_Key'
+  --data-binary '{ 
+    "eventType": "conversion",
+    "eventName": "product added to cart",
+    "indexUid": "products",
+    "objectID": "0",
+    "position": 0
+  }'
+```
+
+It is not possible to associate multiple `conversion` events with the same search.
+
+#### The `conversion` event object
+
+The `conversion` event must deliver an object with the following fields:
+
+```js
+{
+  "eventType": "conversion",
+  "eventName": "product added to cart",
+  "indexUid": "products",
+  "objectID": "0",
+  "position": 0
+}
+```
+
+- `eventType`: indicates this is a `conversion` event
+- `eventName`: an arbitrary string describing the event
+- `indexUid`: the document's index
+- `objectID`: the document's primary key
+- `position`: the document's position in the list of search results
+
+## Understanding the analytics interface
```

Comment:
can we opt for something more optimized for in-docs search and google rankings?

top of my head I have "Search analytics metrics", but there might be other viable alternatives

Reply:
I'm thinking this whole section might be better off as a separate document. I think it was in large part responsible for @qdequele's impression the guide is unfocussed and meandering.

Reply:
I think it would make sense too. My initial idea was to have maybe an "analytics" section in the docs with 2 pages, one being dedicated to the "sending events" kinda?